package.json

```diff
@@ -45,9 +45,9 @@
     "estraverse": "^4.2.0",
     "esutils": "^2.0.2",
     "file-entry-cache": "^1.1.1",
-    "glob": "^7.0.3",
+    "glob2": "^7.0.3",
```

Comment:
@gyandeeps ur right, then get this pr hold for now.

But on the other hand, the code of glob is really Mm.. Ah! Oh! Er!, not that pluggable, and it is not that easy to be monkeypatched. `glob` is highly relied on private members of `minimatch`, and has a lot of duplicate slices(copy & paste) of codes between `glob` and `glob.sync`.

But we know, usage makes proven, which is contradictory somehow.